admin-ui/README.md

```
@@ -1,112 +1,77 @@
 ```
 
-2. Environment configuration
-
-Set the env `NODE_ENV` to `development`.
-
-```
-export NODE_ENV=development
-```
-
-Create a file name `.env.development` with following contents.
+2. Create `.env.development`:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)